networks/suzuka/suzuka-client/src/tests/hey-partners/tests/typescript-sdk/main.test.ts

```diff
@@ -22,17 +22,20 @@ const bob = Account.generate();
 
 const amount = 100000000;
 
-async function transact(signer: any, func: `${string}::${string}::${string}`, typeArgs: string[], args: any[]) {
-    const transaction = await aptos.transaction.build.simple({
+async function buildTransaction(signer: any, func: `${string}::${string}::${string}`, typeArgs: string[], args: any[]) {
```

Comment:
This test comes for our partners, so we would like to keep it as is. 

This also unnecessarily rewraps the method from the Aptos CLI and does not perform any clear "optimization." If anything, you will be adding another stack frame.